model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -296,6 +296,14 @@ message ProcessBundleRequest {
   // (Optional) A list of cache tokens that can be used by an SDK to reuse
   // cached data returned by the State API across multiple bundles.
   repeated CacheToken cache_tokens = 2;
+
+  // (optional) Elements to be processed with the bundle. Either all or
+  // none of the bundle elements should be included in the ProcessBundleRequest.
+  // This elements embedding is to achieve better Fn API efficiency for bundles
+  // that contain only small amount of data and are cheap to be processed on the
+  // SDK harness side. See more at
+  // https://s.apache.org/beam-fn-api-control-data-embedding.
+  Elements elements = 3;
```

Comment:
```suggestion
  // (optional) Elements to be processed within the bundle. Either all or
  // none of the elements should be included in the ProcessBundleRequest.
  // This embedding is to achieve better efficiency for bundles
  // that contain only small amounts of data. See more at
  // https://s.apache.org/beam-fn-api-control-data-embedding.
  Elements elements = 3;
```

Reply:
Please add to the comment something like: `This field will can be set if the SDK declares that they support the "...." capability.`

Reply:
Done.

Reply:
What do you think of my first comment which updated the existing text as well?

Reply:
I guess I missed one of the change and thought I've updated both.